lib/inspec/reporters/json.rb

```diff
@@ -60,7 +60,8 @@ def profile_controls(profile)
         control = {
           id: c[:id],
           title: c[:title],
-          desc: c[:desc],
+          desc: c&.dig(:descriptions, :default),
```

Comment:
Hey, you don't have to use both '&.' and 'dig'. We know c is defined, and a hashlike thing, so you can certainly call `.dig` .  Dig will handle any nil steps on its own.